File: App.Core/Models/Person.cs
using App.Core.Interfaces;
using System;

namespace App.Core.Models
{
    public class Person : IEntity<Guid>
    {
        public Guid Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string EmailAddress { get; set; }

        public DateTimeOffset DateCreated { get; set; }
    }
}

File: Fabric.Web/wwwroot/js/Models/Person.js
export default class {
    constructor(model){
        this.firstName = model.firstName || '';
        this.lastName = model.lastName || '';
        this.emailAddress = model.emailAddress || '';
        this.id = model.id || null;
        this.dateCreated = model.dateCreated ? moment(model.dateCreated) : null;
    }

    get dateCreatedFormatted(){
        return this.dateCreated ? this.dateCreated.format('LLLL') : '';
    }
}